### PR TITLE
Replace nuget.org feed with MUX-Dependencies feed

### PR DESCRIPTION
**Pull request by kmahone**
It is preferable to not depend on nuget.org directly for build time dependencies. I have created a MUX-Dependencies feed that we can use instead.

### CODE REVIEW

File: nuget.config [hunk 1]
```diff
@@ -2,8 +2,11 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
+
+    <!-- do not add new sources to this file. If new packages are required, push them to one of these feeds. -->
     <add key="dotnetfeed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="MUX-Dependencies" value="https://pkgs.dev.azure.com/ms/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json" />
+
     <add key="localpackages" value="localpackages" />
```
Comment (jevansaks) [inline nuget.config]: remind me what this was for? Do we still need it?

Comment (kmahone) [reply nuget.config]: This allows you to test nupkg files locally without pushing them to a feed. You can just put the nupkg file in the 'localpackages' dir.

**Comment by kaiguo** [comment]
Just curious, are we hosting it on our own server? Or is there some public service we can use?

**Comment by kmahone** [comment]
> Just curious, are we hosting it on our own server? Or is there some public service we can use?

We are hosting it in our Azure DevOps project. Hosting public feeds is a new feature that they now support.

**Pull request merged**